ci/docker/Dockerfile.build.ubuntu

```diff
@@ -127,6 +127,11 @@ RUN /work/ubuntu_julia.sh
 COPY install/ubuntu_perl.sh /work/
 RUN /work/ubuntu_perl.sh
 
+# MXNetJS nightly needs emscripten for wasm
+COPY install/ubuntu_emscripten.sh /work/
```

Comment:
`wasm` backend is upstream in llvm since version 8. We have version 10 available in this docker image. So you could delete all the `emscripten-fastcomp` from this script, as `emscripten-fastcomp` is deprecated and replaced by the upstream llvm support.

Reply:
If I only clone emscripten from emscripten-core (with clang and llvm installed) it says:
CRITICAL:root:fastcomp in use, but LLVM has not been built with the JavaScript backend as a target, llc reports:
Not sure why is that

Reply:
My understanding is, that you don't want to use fastcomp backend but rather the native llvm backend. Reference: https://v8.dev/blog/emscripten-llvm-wasm

However, if you find it's difficult to figure out how to do that, it's also OK to keep the current setup for now and fix it later.

Reply:
> My understanding is, that you don't want to use fastcomp backend but rather the native llvm backend. Reference: https://v8.dev/blog/emscripten-llvm-wasm
> 
> However, if you find it's difficult to figure out how to do that, it's also OK to keep the current setup for now and fix it later.

Thanks for the pointer. Seems that I also need emsdk for this.

Reply:
The support for SSE is removed in newer emscripten, when I use the latest emscripten it says mmintrins.h not found, even though the makefile disables SSE.
See: https://github.com/emscripten-core/emscripten/issues/8576